hazelcast/src/test/java/com/hazelcast/topic/TopicTest.java

```diff
@@ -627,4 +629,76 @@ public void onMessage(Message msg) {
         assertEquals(1000, stats.getPublishOperationCount());
         assertEquals(2000, stats.getReceiveOperationCount());
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testTopicMultiThreading() throws Exception {
```

Comment:
this depends on the machine where tests are runing. on my machine it states
`Finished Running Test: testTopicMultiThreading in 8,576 seconds.`
the test code is the same as in `testTopicLocalOrder` and `testTopicGlobalOrder` and takes similair time. But these tests are not anotated.